browser/layout/partials/ctm.html

```diff
@@ -2,7 +2,20 @@
 var classes = document.documentElement.className; {{!note: once we ditch support for ie9 we can just use classlist}}
 classes = classes.replace(/\bno-js\b/, 'js'); {{!note: needed for JS that's run out of the scope of the bootstrap (inline onclicks, etc)}}
 
-window.cutsTheMustard = (typeof Function.prototype.bind !== 'undefined' && typeof document.documentElement.dataset === 'object' && ('withCredentials' in new XMLHttpRequest()));
+var getRandomValuesAvailable;
+try {
+	getRandomValuesAvailable = 'getRandomValues' in window.crypto
+} catch (e) {
+	getRandomValuesAvailable = 'getRandomValues' in window.msCrypto
```

Comment:
Should this also be in a try/catch to prevent errors on non-IE browsers which don't support `.crypto`?

Alternatively avoid the whole nasty try catch with:
```
var crypto = window.crypto || window.msCrypto;
if (crypto) {
   getRandomValuesAvailable = 'getRandomValues' in crypto;
}
```
maybe?